Source/FolioReaderPage.swift

```diff
@@ -656,7 +663,11 @@ extension UIWebView {
     // MARK: - Create and show menu
     
     func createMenu(options options: Bool) {
-        isShare = options
+		guard readerConfig.useReaderMenuController else {
+			return
```

Comment:
The `createMenu` will be only called if you remove the `guard` from `canPerformAction` as you did. What I said is to not remove, but instead of `return false` return the super call, and leave the rest of the function, so it will call super and not create the custom menu, only the default one.